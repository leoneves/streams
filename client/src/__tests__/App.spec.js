import React from 'react'
import { shallow } from 'enzyme'

import App from '../components/App'

describe(App, () => {
  context('with App reders Corret', () => { 
    it('Should match with snapshot', () => {
      const wrapper = shallow(<App />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
