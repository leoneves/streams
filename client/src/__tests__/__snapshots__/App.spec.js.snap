// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App with App reders Corret Should match with snapshot 1`] = `
<div
  className="ui container"
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <div>
      <Header />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/streams/new"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/streams/edit/:id"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/streams/delete/:id"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/streams/show"
      />
    </div>
  </Router>
</div>
`;
